src/pages/resources/migration/ios/tabs/migrate-to-3x.md

```diff
@@ -0,0 +1,50 @@
+<Variant lang="obj-c" repeat="1"/>
+
+```objc
+@import AEPCore;
+@import AEPSignal;
+@import AEPLifecycle;
+@import AEPIdentity;
+@import AEPUserProfile;
+@import AEPServices;
+@import AEPAssurance;
```

Comment:
the cocoapods instructions don't include Assurance, let's remove it from imports and registration also